budget-vs-actual/provincial/budget-vs-actual-provincial/pipeline-spec.yaml

```
@@ -0,0 +1,271 @@
+budget-vs-actual-provincial:
+  title: Provincial budgeted and actual provincial expenditure 2019
```

eave the title the same - we're just adding data - it's an ever-growing file.
```suggestion
  title: Provincial budgeted and actual expenditure
```